rasa/graph_components/validators/default_recipe_validator.py

```diff
@@ -48,7 +49,7 @@
 import rasa.shared.utils.io
 
 
-# TODO: replace these once the Recipe is merged
+# TODO: Can we replace this with the registered types from the regitry?
```

Comment:
Unfortunately this doesn't work as expected:
* `EntitySynonymMapper` is a `EntityExtractor` in the recipe but not part of this
* `DIETClassifier` is `IntentClassifier` in the recipe but a `EntityExtractor` here